app/helpers/injection_helper.rb

```diff
@@ -154,7 +154,6 @@ def default_enterprise_query
   end
 
   def enterprise_injection_data
-    @enterprise_injection_data ||= OpenFoodNetwork::EnterpriseInjectionData.new
-    { data: @enterprise_injection_data }
+    @enterprise_injection_data ||= { data: OpenFoodNetwork::EnterpriseInjectionData.new }
```

Comment:
This is (and was) slightly confusing, because the method `enterprise_injection_data` refers to a hash, rather than the EnterpriseInjectionData instance directly. 
But all good 👍